source/vibe/web/rest.d

```diff
@@ -1,10 +1,226 @@
-/**
-	Automatic REST interface and client code generation facilities.
+// TODO: sub-interfaces
```

Comment:
Note: This comment will obviously go. It's just a personal reminder I didn't document behaviour for sub-interfaces (e.g. it's nowhere documented why getters for sub-interface needs to be parameter-less... @Dicebot ? :P )